sys/src/cmd/stats.c

```diff
@@ -277,9 +277,12 @@ mkcol(int i, int c0, int c1, int c2)
 void
 colinit(void)
 {
-	mediumfont = openfont(display, "/lib/font/bit/pelm/latin1.8.font");
+
```

Comment:
This is kinda gross. We have to do this to *every* *program* that wants a font that doesn't suck?

There has to be a better way.